lib/src/controls/navigation/tree_view.dart

```diff
@@ -688,12 +701,21 @@ class _TreeViewItem extends StatelessWidget {
           return FocusBorder(
             focused: states.isFocused,
             child: Stack(children: [
+              // Indentation and selection indicator for single selection mode.
               Container(
                 height: selectionMode == TreeViewSelectionMode.multiple
                     ? 28.0
                     : 26.0,
                 padding: EdgeInsetsDirectional.only(
-                  start: 12.0 + item.depth * _whiteSpace,
+                  start: selectionMode == TreeViewSelectionMode.multiple
+                      ? !narrowSpacing
+                          ? item.depth * _whiteSpace * 3
```

Comment:
I believe we could add a top-level const for the narrow spacing factor. What do you think?

Reply:
I'm not sure if I understand what you would like to do. 
```
selectionMode == TreeViewSelectionMode.multiple ?
   !narrowSpacing ? 
      item.depth * _whiteSpace * 3          // multi-select + wide spacing: add a padding of 24px per depth level
      :                                     // to have a little gap between the checkboxes
      item.depth * (_whiteSpace * 2 + 4)    // multi-select + narrow spacing: add a padding of 20px per depth level
   :                                        // to have the child's checkbox' start line up with the parent's checkbox' end
   !narrowSpacing ? 
      item.depth * _whiteSpace * 2          // single-select + wide spacing: add a padding of 16px per depth level
      :                                     // to have a 8px gap between the expand icons
      item.depth * _whiteSpace              // single-select + narrow spacing: add a  padding of just 8px per depth level
```

For which of these factors would you like to add a const?